### PR TITLE
fix '[Migrate ASP Tests to TestCafe OS] Hammerhead doesn't inject scripts in iframe on any page of ASPxSpreadsheet demo' (close #815)

### LINKED ISSUE #815: [Migrate ASP Tests to TestCafe OS] Hammerhead doesn't inject scripts in iframe on any page of ASPxSpreadsheet demo
e.g. on http://builder03/Demos-15.2/ASPxSpreadsheetDemos/SampleDocuments/ProfitAndLoss.aspx
It reproduced with hammerhead-playground
We need this fix for ASPx Tests for git hub vesion TestCafe. Please fix it ASAP.

### PR DESCRIPTION
**Pull request by miherlosev**
@churkin @LavrovArtem @helen-dikareva

### CODE REVIEW

**Comment by helen-dikareva** [comment]
Let @VasilyStrelyaev check comment

File: src/client/utils/dom.js [hunk 1]
```diff
@@ -363,11 +363,12 @@ export function isIframeWithoutSrc (iframe) {
     var parentWindowLocation       = parsedParentWindowLocation ? parsedParentWindowLocation.destUrl : parentWindowWithSrc.location.toString();
 
     if (iframeDocumentLocationHaveSupportedProtocol) {
-        // NOTE: After document.open is called for a same-domain iframe,
-        // the iframe window location becomes equal to the location of the parent window with a src.
-        var parsedIframeSrcLocation = urlUtils.parseUrl(iframeSrcLocation);
+        // NOTE: In IE, after document.open is called for a same-domain iframe or iframe with a javascript src,
+        // the iframe window location becomes equal to the location of the parent window with the src.
```
Comment (VasilyStrelyaev) [inline src/client/utils/dom.js]: These article issues... while I agreed not to insert missing articles, here there is a redundant article, which looks weirder than missing ones. So fix please.
```
        // NOTE: In IE, after document.open is called for a same-domain iframe or an iframe with a javascript src,
        // the iframe window location becomes equal to the location of the parent window with src.
```

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR
@churkin @LavrovArtem

**Pull request merged**